nixos/modules/services/home-automation/mycroft.nix

```diff
@@ -0,0 +1,105 @@
+{ config
+, lib
+, pkgs
+, ...
+}:
+
+with lib;
+
+let
+  cfg = config.services.mycroft;
+
+  format = pkgs.formats.json {};
+in
+{
+  meta.maintainers = teams.mycroft.members;
+
+  options.services.mycroft = {
+    enable = mkEnableOption "Mycroft, the private and open voice assistant";
+  };
+
+  config = mkIf cfg.enable {
```

Comment:
```suggestion
  config = mkIf cfg.enable {
     services.mycroft.settings = {
         data_dir = "/var/lib/mycroft";
     };
```

Reply:
I think it's better to make `settings` a freeform type and add an option with a default value there.